conjure/servicewriter.go

```diff
@@ -119,7 +119,11 @@ func astForEndpointParameterArg(argDef *types.EndpointArgumentDefinition, isServ
 		// special case: "binary" types resolve to []byte, but this indicates a streaming parameter when
 		// specified as the request argument of a service, so use "io.ReadCloser".
```

Comment:
Update comment to indicate we're using `*io.ReadCloser` if it's optional

Reply:
Done.